earn/src/App.tsx

```diff
@@ -179,18 +178,6 @@ function App() {
   const provider = useProvider({ chainId: activeChain.id });
 
   const value = { activeChain, setActiveChain };
-  const twentyFourHoursAgo = Date.now() / 1000 - 24 * 60 * 60;
-  const BLOCK_QUERY = gql`
-  {
-    blocks(first: 1, orderBy: timestamp, orderDirection: asc, where: {timestamp_gt: "${twentyFourHoursAgo.toFixed(
-      0
-    )}"}) {
-      id
-      number
-      timestamp
-    }
-  }
-  `;
```

Comment:
We weren't using this anywhere. It's in this PR because I was going to use it for something, and when I decided not to, I did a Find References and saw it's vestigial.